frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```
@@ -370,9 +370,9 @@ action.GitMachete.FastForwardMergeCurrentToParentAction.GitMacheteToolbar.text=F
 action.GitMachete.FastForwardMergeCurrentToParentAction.description=Fast-forward merge current branch to parent
 
 
-action.GitMachete.TraverseFromCurrentAction.text=Git Machete: Traverse from Current Branch\u2026
-action.GitMachete.TraverseFromCurrentAction.GitMacheteToolbar.text=Traverse from Current Branch\u2026
-action.GitMachete.TraverseFromCurrentAction.description=Traverse from current branch
+action.GitMachete.TraverseFromFirstAction.text=Git Machete: Traverse from First Branch\u2026
+action.GitMachete.TraverseFromFirstAction.GitMacheteToolbar.text=Traverse from First Branch\u2026
+action.GitMachete.TraverseFromFirstAction.description=Traverse from first branch
```

```suggestion
action.GitMachete.TraverseFromFirstAction.text=Git Machete: Traverse from the First Branch\u2026
action.GitMachete.TraverseFromFirstAction.GitMacheteToolbar.text=Traverse from the First Branch\u2026
action.GitMachete.TraverseFromFirstAction.description=Traverse from the first branch
```
Idk, `first` without a preceding `the` looks weird to me